Amazon.Ion.ObjectMapper/IonObjectSerializer.cs

```diff
@@ -35,36 +48,19 @@ public override object Deserialize(IIonReader reader)
                 FieldInfo field;
                 if (property != null)
                 {
-                    if (IsReadOnlyProperty(property))
+                    var deserialized = new object();
+                    if (this.DeserializeProperty(property, reader, ionType, ref deserialized))
```

Comment:
When we return a boolean to indicate success, the method name should be prefixed with `Try`, so this would become `TryDeserializeProperty`.

Reply:
Done.